new/LIT6274MGFireHouse.xml

```diff
@@ -0,0 +1,106 @@
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="SH" when="2020-11-30">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+
+                <listBibl type="editions">
+                    <bibl>
+                        <ptr target="bm:Arras1953GeorgiusText"/>
+                        <citedRange unit="page">30–31 (28th miracle)</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:Arras1953GeorgiusTrans"/>
+                        <citedRange unit="page">17 (28th miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+
+                <listBibl type="secondary">
+                    <bibl>
+                        <ptr target="bm:Strelcyn1974CatJRylands"/>
+                        <citedRange unit="page">36</citedRange>
+                        <citedRange unit="item">Ms 12, II.29 (28th miracle)</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:GrebTiss1935Codices"/>
+                        <citedRange unit="page">723</citedRange>
+                        <citedRange unit="item">Vat 233, 3.28 (28th miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT6274MGFireHouse"
+                        passive="LIT4006Taamme"/>
+                </listRelation>
+
+            </div>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="BDLaethb4"/>.</note>
+                    <ab>፳፰ ተአምሪሁ፡ <gap reason="ellipsis" resp="SH"/> ወኮነ፡ በ፩እመ<supplied
```

Comment:
shall we leave a space after ፩ in this case? 
PS also line below, ለ፩ብእሲ፡

Reply:
@thea-m : do we add a space in those instances?

Reply:
I would tend towards yes following our existing policy on spaces after a numeral, but don't remember whether we discussed cases as this

Reply:
Let's add this to our meeting agenda

Reply:
I am absoluely fine with that